docs/articles/documentation/getting-started/index.md

```diff
@@ -0,0 +1,165 @@
+
+Test code is placed within the `test` function.
+
+### Performing Actions on Page
+
+Every test should be capable of interacting with a page content. To perform user actions, TestCafe provides a number of [actions](../test-api/actions.md): `click`, `hover`, `typeText`, `upload`, etc.
+They can be called in a chain.
+
+The following fixture contains a simple test for the [http://testcafe.devexpress.com/Example](http://testcafe.devexpress.com/Example) sample page.
+The test types a developer name into a text editor and then clicks the Submit button.
+You can try running this test against the sample page by yourself.
+
+```js
+fixture `Getting Started - Test 1`
+    .page('http://testcafe.devexpress.com/Example');
+
+test('MyTest', async t => {
+    await t
+        .typeText('#Developer_Name', 'John Smith')
+        .click('#submit-button');
+});
+```
+
+All test actions are implemented as asynchronous functions of the test controller object `t`. This object is used to access test run API.
+To wait for actions to be complete, use the `await` function when calling these actions or action chains.
+
+### Observing Page State
+
+TestCafe allows you to perform custom actions on a page element and observe the element's state.
+For this purpose, it provides the `Hybrid` function.
+You wrap your code into this function, and it is executed on the client side, thus giving you direct access to DOM elements.
+
+```js
+const actualText = Hybrid(() => document.querySelector('#article-header').text);
+```
+
+For example, you can use the hybrid function to obtain an element state to further check it in [assertions](#assertions).
+
+For more information, see [Executing Client Code](../test-api/executing-client-code.md).
+
+### Assertions
+
+A functional test should check the result of actions performed.
+For example, clicking the Submit button on the sample web page opens a "Thank you" page whose header addresses a user by the entered name.
+To check if the name is correct, you have to add an assertion to the test.
+
+You can use assertions from Node's built-in [assert](https://nodejs.org/api/assert.html) module or from any third-party assertion library.
+Before calling assertions, make sure an assertion library is installed into your tested project.
+
+The following test demonstrates how to use an assertion from [Chai Assertion Library](http://chaijs.com/api/bdd/).
+Before running the test, install the assertion library: go to the tested project directory (*my-project*) and run `npm install chai`.
+
+```js
+fixture `Getting Started - Test 2`
+    .page('http://testcafe.devexpress.com/Example');
+
+test('MyTest', async t => {
+    await t
+        .typeText('#Developer_Name', 'John Smith')
+        .click('#submit-button');
+
+    const actualText = Hybrid(() => document.querySelector('#article-header').text);
+
+    expect(await actualText.to.equal('Thank You, John Smith!');
```

Comment:
Also, missing `)` after `actualText` and `actualText` is not called. Also it makes sense use `t.eval` here:
```js
const actualText = await t.eval(() => document.querySelector('#article-header').text);

expect(actualText).to.equal('Thank You, John Smith!');
```